addons/sale_timesheet/views/project_portal_templates.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
     <template id="portal_timesheet_table_inherit" inherit_id="hr_timesheet.portal_timesheet_table">
-        <xpath expr="//thead/tr/th[@t-if='is_uom_day']" position="before">
+        <xpath expr="//thead//tr//th[hasclass('text-end')]" position="before">
```

Comment:
Maybe not the best way to identify a tag. I think it is better to put an id on it :slightly_smiling_face: